EFCore.InMemory.HierarchyId.Test/EFCore.InMemory.HierarchyId.Test.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0;netcoreapp3.1</TargetFrameworks>
+    <IsPackable>false</IsPackable>
+    <AssemblyName>EntityFrameworkCore.InMemory.HierarchyId.Test</AssemblyName>
+    <RootNamespace>Microsoft.EntityFrameworkCore.InMemory</RootNamespace>
+    <Features>strict</Features>
+    <LangVersion>latest</LangVersion>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\key.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="5.0.0-rc.2.20475.6" />
```

Comment:
strange that Design is needed for InMemory.

Reply:
There was no need for that, removed in https://github.com/efcore/EFCore.SqlServer.HierarchyId/pull/33/commits/ceb49274fbfbbfbe7fba9713094f5ee9b0871f0c